3-platform/order-ms/app/src/main/java/com/core/platform/orderms/app/OrderUseCase.java

```diff
@@ -0,0 +1,21 @@
+package com.core.platform.orderms.app;
```

Comment:
bad naming of package -> it should be sum of groupId + artifactId. 
In java package names is reversed name of company, e.g. for mate.academy.com -> package name will `com.mate.academy`.
```suggestion
package com._6core.platform.orderms.app;
```